src/pages/home/report/ContextMenu/ReportActionContextMenu.ts

```diff
@@ -64,8 +64,7 @@ function hideContextMenu(shouldDelay?: boolean, onHideCallback = () => {}) {
         return;
     }
     if (!shouldDelay) {
-        contextMenuRef.current.hideContextMenu(onHideCallback);
-
+        onHideCallback();
```

Comment:
@teneeto  Thanks for the PR.
Can you please share some information on how this change helps in resolving the issue? 
Also, this code has not been changed for quite some time. So, I am curious to know what recent changes could have triggered this issue.